src/views/developers/l10n.json

```diff
@@ -32,13 +40,19 @@
   "developers.designPrinciplesTinkerBody": "We believe that the learning process is inherently iterative. Tinkerers start by exploring and experimenting, then revising and refining their goals and creations. To support this style of interaction, we design our interfaces to encourage quick experimentation and rapid cycles of iteration.",
-  "developers.joinBody": "We're a diverse group of educators, designers, and engineers, who work together in a playful, creative environment full of LEGO bricks, craft materials, and maker tools. We strongly value diversity, collaboration, and respect in the workplace. If you're interested in joining us, take a look at our open positions on our <a href=\"/jobs\">Jobs Page</a>, or send us an email at <a href=\"mailto:jobs+developers@scratch.mit.edu\">jobs+developers@scratch.mit.edu</a>.",
-  "developers.donateIntro": "We are pleased to provide Scratch free of charge. If you enjoy using Scratch, please consider <a href=\"https://secure.donationpay.org/scratchfoundation/\">making a donation to support Scratch</a>. Donations of any size are appreciated.",
+  "developers.joinBodyJobsLinkText": "Jobs Page",
+  "developers.joinBodyEmailLinkText": "jobs+developers@scratch.mit.edu",
+  "developers.joinBody": "We're a diverse group of educators, designers, and engineers, who work together in a playful, creative environment full of LEGO bricks, craft materials, and maker tools. We strongly value diversity, collaboration, and respect in the workplace. If you're interested in joining us, take a look at our open positions on our {jobsPageLink}, or send us an email at {emailLink}.",
+  "developers.donateIntroLinkText": "making a donation to support Scratch",
+  "developers.donateIntro": "We are pleased to provide Scratch free of charge. If you enjoy using Scratch, please consider {donateLink}. Donations of any size are appreciated.",
   "developers.donateBody": "Your donation to the Scratch Foundation will be used to support future development of Scratch software and the Scratch website.",
   "developers.donateThanks": "Thanks for supporting Scratch!",
   "developers.partnersIntro": "The creation and maintenance of this open source code would not be possible without generous technical and financial support from our partners:",
   "developers.faqAboutTitle": "Where can I learn more about Scratch?",
-  "developers.faqAboutBody": "Scratch is a free programming language and online community where young people can create their own interactive stories, games, and animations. Scratch is a project of the <a href=\"https://www.media.mit.edu/groups/lifelong-kindergarten/overview\">Lifelong Kindergarten</a> Group at the <a href=\"http://media.mit.edu/\">MIT Media Lab</a>. You can learn more about Scratch <a href=\"/about\">here</a>.",
+  "developers.faqAboutBodyLLKLinkText": "Lifelong Kindergarten",
+  "developers.faqAboutBodyMITLinkText": "MIT Media Lab",
+  "developers.faqAboutBodyAboutLinkText": "here",
```

Comment:
I think there are still multiple places where we link to the word 'here'. To make it easier on translators, let's just have one `"developers.hereLinkText": "here"` and use that everywhere.